wix-media-v2/wix-media-v2/guides/media-intro.md

```diff
@@ -31,53 +35,15 @@ To use the wix-media API, import `wixMedia` from the `wix-media.v2` module.
-  There are 3 types of system folders:
-  * **Root Folders:** Main categories of folders in the Media Manager. 
-    * `MEDIA_ROOT`: Contains all files and folders in the Media Manager's 'Site Files' tab in the UI.
-
-      >**Note:** The `MEDIA_ROOT` system folder is different from the `media-root` folder of the Media Manager. The `MEDIA_ROOT` system folder contains all media in all levels of the 'Site Files' tab in the UI, while the `media-root` folder of the Media Manager only contains the media in the root level of the 'Site Files' tab in the UI.
-
-    * `TRASH_ROOT`: Contains all files and folders in the Media Manager's 'Trash' tab in the UI.
-    * `VISITOR_UPLOADS_ROOT`: Contains all files and folders created by site visitors or members.
-
-      >**Note:** The `VISITOR_UPLOADS_ROOT` folder is located in the 'Site Files' tab in the UI. However, it is its own root folder and doesn't return when searching in the `MEDIA_ROOT` folder.
-
-  * **Virtual Folders:** Special categories of folders in the Media Manager that are hidden from the UI and can't be deleted.
-    * `MOBILE_UPLOADS`: Contains all media uploaded using mobile devices. 
-    * `PURCHASED_ITEMS`: Contains all media imported using a purchase flow.
-
-
-  * **Other Folders:** 
-    * `VIDEO_MAKER`: Contains all videos created using Wix Video Maker. Located in the 'Site Files' tab in the UI.  
-
-                                                                                                                                                                          
-* **File Assets** Wix Media files are optimized for web use. When a file is imported or uploaded to the Media Manager, it is processed and may produce several variations of the file for use in different circumstances.    
-  For example:   
-    * A video file can have different resolutions and formats.
-    * An audio file can have different formats and qualities.
-    * Video or audio files can have a preview asset containing only a portion of the file.
-  
-  With the [Generate File Download Url](wix-media-v2/files/generatefiledownloadurl), you can use the `assetKeys` parameter to download different assets of a file.  
-
-* **Media Types:** Supported media types include the following:
-  * Image
-  * Video
-  * Audio
-  * Document
-  * Vector
-  * Archive
-  * Model 3D
-  Learn more about [media file types and size limits](https://support.wix.com/en/article/wix-media-supported-media-file-types-and-file-sizes).
\ No newline at end of file
+* **Root Folders:**
+  * `MEDIA_ROOT`: Contains all files and folders in the Media Manager's 'Site Files' tab in the UI.
+    >**Note:** The `MEDIA_ROOT` system folder is different from the `media-root` folder of the Media Manager. The `MEDIA_ROOT` system folder contains all media in all levels of the 'Site Files' tab in the UI, while the `media-root` folder of the Media Manager only contains the media in the root level of the 'Site Files' tab in the UI.
```

Comment:
I think you need a blank line above and below this note so everything has some breathing room.